adoc/deployment-preparation.adoc

```diff
@@ -121,7 +121,7 @@ Setting up a load balancer is mandatory in any production environment.
 
 {productname} requires a load balancer to distribute workload between the deployed
 master nodes of the cluster. A failure tolerant {productname} cluster will
-always use more than one load balancer since that becomes a "single point of failure".
+always use more than one master node since that becomes a "single point of failure".
```

Comment:
This was intended, ideally for a real HA environment, aside from several control planes the user should use several load balancers, otherwise if they have a single LB it becomes a single point of failure. Maybe we can rephrase this to state both things: `A failure tolerant {productname} cluster will always use more than one control plane node as well as more than one load balancer, so there is not a single point of failure`, or something similar?

Reply:
thanks, fixed